### DISCUSSION TITLE
Looking for experienced crypto developers.

### DISCUSSION THREAD

**Discussion by Hawkern** [Homebrew/discussions#1251] [Everyday usage]
Hey, i'm looking for one or a few at highest experienced crypto developers, maybe you can point me in the right direction?
I got a little idea i would like to bring to life, but i don't have the experience in that particular area. 

So if anyone here is or knows one who is experienced in crypto-developing holla at me please.

Thanks in advance

Best regards, 
H

**Comment by SMillerDev** [comment]
Why not try contacting people at crypto projects like openssl/gnutls? Homebrew writes a package manager in ruby, I doubt there is much overlap in skills.
